demisto_sdk/commands/content_graph/objects/base_playbook.py

```diff
@@ -206,7 +206,7 @@ def _client_upload_method(cls, client: demisto_client) -> Callable:
         return client.import_playbook
 
     def save(self):
-        super().save()
+        super().save(fields_to_exclude=["tasks"])
```

Comment:
why dont we want this to be saved?

Reply:
we do, we just save it by parsing the object itself as part of the base_playbook save and not the super.